src/uct/api/uct.h

```diff
@@ -1788,9 +1789,10 @@ UCT_INLINE_API ucs_status_t uct_ep_tag_eager_short(uct_ep_h ep, uct_tag_t tag,
 
 /**
  * @ingroup UCT_TAG
- * @brief Send tagged message using bcopy eager protocol.
+ * @brief Bcopy eager tagged-send operation.
  *
- * Eager protocol means that the whole data is sent immediately to the peer.
+ * This routine sends a message using buffered copy eager protocol. Eager
```

Comment:
Agree, that is why we need to have one place which describes what is "short" "bcopy" and "zcopy", then each API would describe the operations itself, and not elaborate much (but refer to) the data type specification. what do you think?